### PR TITLE
WINDUP-447: NPE halts Windup execution

### PR DESCRIPTION
**Pull request by jsight**
https://issues.jboss.org/browse/WINDUP-447

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Merged build triggered.

**Comment by windupgithubbot1** [comment]
Merged build triggered.

File: reporting/impl/src/main/java/org/jboss/windup/reporting/freemarker/ProjectModelToApplicationIndexMethod.java [hunk 1]
```diff
@@ -60,6 +60,10 @@ public Object exec(@SuppressWarnings("rawtypes") List arguments) throws Template
             throw new TemplateModelException("Error, method expects one argument (ProjectModel)");
         }
         StringModel stringModelArg = (StringModel) arguments.get(0);
+        if (stringModelArg == null)
+        {
+            throw new IllegalArgumentException("FreeMarker Method " + NAME + " called with null project model");
```
Comment (lincolnthree) [inline reporting/impl/src/main/java/org/jboss/windup/reporting/freemarker/ProjectModelToApplicationIndexMethod.java]: Should this print which template this was found in?

Comment (jsight) [inline reporting/impl/src/main/java/org/jboss/windup/reporting/freemarker/ProjectModelToApplicationIndexMethod.java]: That would be great, but I don't think I have any way of knowing that. :(

**Comment by windupgithubbot1** [comment]
Merged build started.

**Comment by windupgithubbot1** [comment]
Merged build finished. Test FAILed.

**Comment by windupgithubbot1** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://10.3.9.192//job/windup-pr-builder/364/
Test FAILed.

**Pull request merged**